Makefile

```diff
@@ -45,23 +45,27 @@ manifests: controller-gen
 .PHONY: fmt
 fmt:
 	go fmt ./...
+	./hack/verify-diff.sh
 
 # Run go vet against code
 .PHONY: vet
 vet:
 	go vet ./...
+	./hack/verify-diff.sh
 
 # Run golangci-lint against code
 .PHONY: lint
 lint: golangci-lint
 	( GOLANGCI_LINT_CACHE=$(PROJECT_DIR)/.cache $(GOLANGCI_LINT) run --timeout 10m )
+	./hack/verify-diff.sh
 
 # Run go mod
 .PHONY: vendor
 vendor:
 	go mod tidy
 	go mod vendor
 	go mod verify
+	./hack/verify-diff.sh
```

Comment:
What about adding a new make target than can run other targets

```
verify-%:
  make $*
  ./hack/verify-diff.sh
```
Then we can have the ability to run `make vendor` without the verify (which would be useful)

Reply:
currently the behaviour is set by OPENSHIFT_CI env variable in the script. If it's true, then we perform verification, otherwise omit it. Isn't it enough for us?
So, on your local machine `make vendor` will update the code and return successfully, but on CI it will report an error if something is wrong.

Reply:
Changing the behaviour based on an env var like that seems very much like magic and is very non-obvious (I will not remember this in 6 months), I would rather we be explicit about having separate tasks. 

As an example, if I wanted to verify there was no diff locally as well, I would have to try and work out what magic env var I have to set, which is not user friendly.

IMO we should be explicit here and have seperate make tasks for verification

Reply:
ah, I got what you mean. I also don't like that variable